h2o-r/h2o-package/R/connection.R

```diff
@@ -801,8 +801,9 @@ h2o.clusterStatus <- function() {
   # md5_check <- readLines(tcon, n = 1)
   # close(tcon)
   md5_file <- tempfile(fileext = ".md5")
-  download.file(md5_url, destfile = md5_file, mode = "w", cacheOK = FALSE, quiet = TRUE)
+  .downloadFile(md5_url, dest = md5_file, mode = "w")
   md5_check <- readLines(md5_file, n = 1L)
+    cat(md5_check)
```

Comment:
please fix the indentation if you want to leave the print out

Reply:
removed